nustyle.css

```diff
@@ -376,3 +384,223 @@ ul.icons2 {
   align-self: center;
   margin-top: 15px;
 }
+
+#form-button:hover {
+  background-color: #f8b791;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#footer-hr {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  #nav-mobile {
+    display: none;
+  }
+
+  #nav-desktop {
+    display: inline;
+    align-self: center;
+  }
+
+  #nav-desktop a {
+    padding-right: 32px;
+    margin: 0;
+    font-family: Inter;
+    font-size: 15px;
+    font-weight: 600;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  header img {
+    height: 18px;
+    padding-bottom: 0;
+    padding-right: 119px;
+  }
+
+  .header-logo {
+    padding-left: 154px;
+    font-size: 20px;
+    padding-bottom: 26px;
+  }
+
+  .container {
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    background-image: url(./assets/Illustration-header.png);
+    background-position: right;
+  }
+
+  .icons {
+    flex-direction: column;
+    padding: 279px 0 21px 36px;
+  }
+
+  .intro {
+    padding-left: 71px;
+  }
+
+  .intro p {
+    font-family: 'Poppins';
+    font-size: 16px;
+    font-weight: 400;
+    width: 50%;
+    margin-bottom: 251px;
+  }
+
+  .intro h1 {
+    padding-top: 192px;
+  }
+
+  .icons li {
+    padding-bottom: 21px;
+  }
+
+  .works {
+    margin: 135px 153.5px 0 153.5px;
+  }
+
+  .main-project {
+    padding: 0;
+  }
+
+  .hr {
+    display: inline-block;
+    content: "";
+    background-color: #212121;
+    height: 1px;
+    width: 61%;
+    padding-right: 30px;
+    margin-left: 20px;
+  }
+
+  .banner {
+    border: none;
+    display: flex;
+    align-items: center;
+  }
+
+  #project {
+    display: grid;
+    grid-template-columns: 600px 400px;
+    gap: 24px;
+  }
+
+  #project-story {
+    width: min-content;
+  }
+
+  #project img {
+    min-height: 400px;
+    min-width: 600px;
+    display: flex;
+    flex-shrink: inherit;
+  }
+
+  .other-projects:hover .hover {
+    display: none;
+  }
+
+  #project h2 {
+    margin-top: 0;
+  }
+
+  #projects-container {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    gap: 29px;
+    margin-top: 24px;
+    margin-bottom: 130px;
+  }
```

Comment:
- [ ] Please you still need to make a little adjustment with the background images. Designing according to the given template can increase the confidence that hiring manager have towards developers; this shows their capability to deliver accurately. Kindly make the necessary adjustment. 👍 

## Your design 

![b5](https://user-images.githubusercontent.com/103386183/199943218-c72c0f88-0777-4947-9ffd-d8c053490587.png)

## Figma Design

![b4](https://user-images.githubusercontent.com/103386183/199943280-dfa96cef-53a7-4c28-b404-8ed36148586b.png)

Reply:
@OyePriscilla I really don't think this is necessary, but I'll do it nevertheless.